R/repo-helper.R

```diff
@@ -0,0 +1,75 @@
+#' Create example repo for fledge demos
+#'
+#' @param open Whether to open the new project.
+#' @param pkg Package name.
+#' @param name Name for DESCRIPTION and git.
+#' @param email Email for DESCRIPTION and git.
+#' @param date String of time for DESCRIPTION and git.
+#' @param tempdir Directory within which to create the mock package folder.
+#' @param news If TRUE, create a NEWS.md file.
+#'
+#' @return
+#' @export
+#'
+create_fledge_mock_repo <- function(open = rlang::is_interactive(),
+                               pkg = "tea",
+                               name = "Kirill M\u00fcller",
+                               email = "mail@example.com",
+                               date = "2020-12-12",
+                               tempdir = withr::local_tempdir(
+                                 pattern = "fledge",
+                                 .local_envir = parent.frame(n = 2)
```

Comment:
I am not entirely convinced there should be a default. If not using `.local_envir`, the project does not exist once open.

Reply:
Can we leave the project in the tempdir after we finish, so that it remains there for the duration of the R session?

Reply:
Ah I like this suggestion, it'll make everything easier :grin: